packages/radio-button/src/presenters/ButtonPresenter.js

```diff
@@ -1,42 +1,24 @@
+/* eslint react/forbid-prop-types: 0 */
+
 import React, { Component } from "react";
-import PropTypes from "prop-types";
-import cx from "classnames";
+import { ThemeContext } from "@hig/theme-context";
+import { cx, css } from "emotion";
 
-import "./ButtonPresenter.scss";
+import stylesheet from "./stylesheet";
 
 export default class ButtonPresenter extends Component {
-  static propTypes = {
-    /**
-     * Checks the radio button
-     */
-    checked: PropTypes.bool,
-    /**
-     * Prevents user actions on the radio button
-     */
-    disabled: PropTypes.bool
-  };
-
-  static defaultProps = {
-    checked: false,
-    disabled: false
-  };
-
   render() {
-    const { checked, disabled } = this.props;
-
-    const iconClasses = cx([
-      "hig__radio-button__wrapper",
-      {
-        "hig__radio-button__wrapper--checked": checked,
-        "hig__radio-button__wrapper--disabled": disabled
-      }
-    ]);
-
     return (
-      <span className={iconClasses}>
-        <span className="hig__radio-button__wrapper__button" />
-        <span className="hig__radio-button__wrapper__button--disabled" />
-      </span>
+      <ThemeContext.Consumer>
+        {({ resolvedRoles }) => (
+          <span
+            className={cx(
+              "radio-button__wrapper",
```

Comment:
do we need to keep this css classname?

Reply:
For styling purposes yes. Otherwise I can't hook into the hover/checked effects easily.